symplyphysics/core/convert.py

```diff
@@ -1,19 +1,20 @@
 from sympy import Expr, sympify, S
-from sympy.physics.units import Quantity as SymQuantity
+# from sympy.physics.units import Quantity as SymQuantity
 
 from .dimensions import assert_equivalent_dimension
 from .symbols.quantities import Quantity
 
 
-def convert_to(value: SymQuantity, target_unit: SymQuantity) -> Expr:
+def convert_to(value: Expr, target_unit: Expr) -> Expr:
```

Comment:
Target unit is under our control. We want to narrow down its type as much as possible. Don't change its type.

Reply:
Okay

Reply:
Hm, good point. Maybe it can be safely converted to SymQuantity?

Reply:
Not sure why. Tried to understand the cause but it seems like that's some quirk of `mypy`. But this is a reason to leave `Expr` to be the type of `target_unit`.

Reply:
Ok, let's keep it this way.